src/main/java/com/Project/accountManager/controllers/AccountController.java

```diff
@@ -0,0 +1,62 @@
+package com.Project.accountManager.controllers;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountUpdateRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import com.Project.accountManager.services.AccountService;
+import com.Project.accountManager.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/account")
+public class AccountController {
+
+    @Autowired
+    AccountService accountService;
+    @Autowired
```

Comment:
you already use Constructure. That's why you don't need to use @Autowried annotation.
<img width="503" alt="Screenshot 2024-05-25 at 08 25 27" src="https://github.com/serhatbsts/accountManager/assets/26492690/4b458282-e29b-4633-81ff-2019c9841eb6">


so the following will be better -_
  ```
private final AccountService accountService;

    public AccountController(AccountService accountService) {
        this.accountService = accountService;
    }
```
Also I am suggest to you research about "why shouldn't use @Autworied" 
https://www.linkedin.com/pulse/you-should-stop-using-spring-autowired-felix-coutinho

Reply:
All right. I'll look into it.